app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProfilesController < ApplicationController
+
+  def show
```

Comment:
Related to my routes comment, I'm thinking maybe we could go with `edit` here instead, and rename the view to match.